src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/SubscriptionPage.java

```diff
@@ -1,6 +1,9 @@
 package com.ericsson.ei.frontend.pageobjects;
```

Comment:
All public functions in this file should have javadoc. This is from earlier PR as I understand.